spec/controllers/account/notifications_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Account::NotificationsController, type: :controller do
+  let!(:user) { create(:user, first_name: 'Bob') }
+  let!(:notification) { create(:notification, notification_type: Notification::NOTIFICATION_TYPES.sample, notificable: user, user_id: user.id) }
```

Comment:
why `sample`? u have `user` as `notificable`
what do u think may happen if `sample` gives a ref to organization?